source/patterns/@aws-solutions-constructs/aws-apigateway-kinesisstreams/lib/index.ts

```diff
@@ -169,11 +169,11 @@ export class ApiGatewayToKinesisStreams extends Construct {
       apiGatewayRole: this.apiGatewayRole,
       apiMethod: 'POST',
       apiResource: putRecordResource,
-      requestTemplate: this.getPutRecordTemplate(props.putRecordRequestTemplate),
-      additionalRequestTemplates: this.getAdditionalPutRecordTemplates(props.additionalPutRecordRequestTemplates),
+      requestTemplate: this.buildPutRecordTemplate(props.putRecordRequestTemplate),
```

Comment:
I don't see the check for invalid props at the top of this constructor?

Once they are implemented across each API Gateway construct I think might see the ability to consolidate them in input_validation.ts? Maybe not, but once they are all implemented we can see.

Reply:
There were a handful of constructs that didn't follow the `allow` property pattern, and so the checks weren't added to those (this being one of them).